src/CONST.ts

```diff
@@ -4253,16 +4253,16 @@ const CONST = {
                         'Then, send your request and wait for that sweet “Cha-ching!” when it’s complete.',
                 },
                 {
-                    type: 'enableWallet',
+                    type: 'addBankAccount',
                     autoCompleted: false,
-                    title: 'Enable your wallet',
+                    title: 'Add personal bank account',
                     description:
-                        'You’ll need to *enable your Expensify Wallet* to get paid back. Don’t worry, it’s easy!\n' +
+                        'You’ll need to add your personal bank account to get paid back. Don’t worry, it’s easy!\n' +
                         '\n' +
-                        'Here’s how to set up your wallet:\n' +
+                        'Here’s how to set up your bank account:\n' +
                         '\n' +
                         '1. Click your profile picture.\n' +
-                        '2. Click *Wallet* > *Enable wallet*.\n' +
+                        '2. *Click Wallet* > *Bank accounts* > *+ Add bank account*.\n' +
```

Comment:
I think the bold should be highlighting a label of action.

```suggestion
                        '2. Click *Wallet* > *Bank accounts* > *+ Add bank account*.\n' +
```